src/docs/asciidoc/developer-manual/develop-cloud-services/initialization-code.adoc

```diff
@@ -0,0 +1,27 @@
+:relative-path: ../../
+include::{docdir}/variables.adoc[]
+
+ZetaPush provides you the possibility to add some initialization code in your _custom cloud service_.
+For this you need to implement the method `onApplicationBootstrap()` in your _custom cloud service_.
+
+.Initialization code
+[source, javascript, linenums]
+----
+import { Bootstrappable } from '@zetapush/core';
+
+export default class implements Bootstrappable {
+  async onApplicationBootstrap() {
+    console.log('Initialization');
+  }
+}
+----
+
+The implementation of the `Bootstrappable` interface is optional but it is a best practice, you just need to implement a method named `onApplicationBootstrap()`.
+Indeed, implementing it avoid you to write wrong implementation of `onApplicationBootstrap()` method.
+
+.Several `onApplicationBootstrap()` implementations
+[NOTE]
+====
+You can implement the `onApplicationBootstrap()` method in several _custom cloud services_.
+A dependency tree will be created to execute all `onApplicationBootstrap()` methods in the proper order (Regarding of the needed dependencies).
```

Comment:
~R~**r**egarding of the dependency injection tree.
We should explain which order onApplicationBootstrap will be executed with a sample and a schema (like Modez did)